### ISSUE TITLE
detailed course progress

### ISSUE THREAD

**Issue by fwitulski** [h-da/geli#592]
# User Story

### As a: 

Student

### I want:

to know my exact progress of the course (which exact tasks have I done)

### so that:

I can monitor better what has been done.

## Acceptance criteria:
- [ ] see which exact tasks have been done
- [ ] see amount of submits/attempts

## Additional info:

------
_Please tag this issue if you are sure to which tag(s) it belongs._

**Comment by grosspersky** [comment]
You mean something like the report style that is currently only available for the teacher or the admin?
![grafik](https://user-images.githubusercontent.com/11839276/38453760-1d684b50-3a5b-11e8-96a8-465b9ad08e60.png)

**Comment by fwitulski** [comment]
Yes it can be simular like this, but I would extend it with the attempts needed. I adjusted the description above-
